jest.config.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Removed jest config file and added to package.json so that if you are running jest, npm test, or just kicking it off vscode launch, it would always be the same configuration.